src/assignment/views/AssignmentEdit.tsx

```diff
@@ -891,7 +891,9 @@ const AssignmentEdit: FunctionComponent<AssignmentEditProps & UserProps> = ({
 
 					<InActivityWarningModal
 						onActivity={updateAssignmentEditor}
-						onExit={releaseAssignmentEditStatus}
+						// AVO-2934: do not release edit status after delete assignment
+						//onExit={releaseAssignmentEditStatus}
+						onExit={() => true}
```

Comment:
shouldn't this still do this if the assignment wasn't deleted?

eg:
if assignment was deleted:
    exit () => true
else 
    releaseAssignmentEditStatus